components/Button/buttonPrimary.style.tsx

```diff
@@ -0,0 +1,20 @@
+import styled from "@emotion/styled";
+import primaryTheme from "../../styles/theme/theme";
+
+const Button = styled('a')({
+  backgroundColor: primaryTheme.colors.secondary,
+  color: primaryTheme.colors.primary,
+  padding: '10px 70px',
+  fontSize: '38px',
+  lineHeight: '48px',
+  cursor: 'pointer',
+  borderRadius: '3px',
+  border: 'none',
+  transition: '0.1s ease-in-out',
+
+  '&:hover, &:focus, &:active': {
+    backgroundColor: primaryTheme.colors.secondaryHover
```

Comment:
As per my comment below since we don't have themes on this project we don't have to use all this nesting. Let's keep it simple for example: colos.secondary

We can have for main colors 2-3 primary colors (primary, secondary, tertiary) and then accents (we don't need too many colors), and some default ones

```
const baseColors = {
  white: '#FFFFFF',
  greymid: '#D2D2D2',
  greydark: '#707070',
  offblack: '#262626',
  black: '#000000',
  success: '#008810',
  error: '#EB0023',
  none: 'transparent',
}

const colors = {
    ...baseColors,
    primary: '#090972',
    secondary: '#F9D516',
    accent1: '#1E90FF',
    accent2: '#0F0D3C',
    accent3: '#638787',
 }
```